packages/apps/cdn-viewer/package.json

```diff
@@ -5,26 +5,25 @@
   "main": "src/index.js",
   "dependencies": {
     "@bentley/icons-generic": "^1.0.13",
-    "@bentley/imodelhub-client": "3.0.0-dev.116",
-    "@bentley/itwin-client": "3.0.0-dev.116",
-    "@bentley/product-settings-client": "3.0.0-dev.116",
-    "@bentley/telemetry-client": "3.0.0-dev.116",
-    "@itwin/appui-abstract": "3.0.0-dev.116",
-    "@itwin/appui-layout-react": "3.0.0-dev.116",
-    "@itwin/appui-react": "3.0.0-dev.116",
-    "@itwin/components-react": "3.0.0-dev.116",
-    "@itwin/core-bentley": "3.0.0-dev.116",
-    "@itwin/core-common": "3.0.0-dev.116",
-    "@itwin/core-frontend": "3.0.0-dev.116",
-    "@itwin/core-geometry": "3.0.0-dev.116",
-    "@itwin/core-i18n": "3.0.0-dev.116",
-    "@itwin/core-markup": "3.0.0-dev.116",
-    "@itwin/core-orbitgt": "3.0.0-dev.116",
-    "@itwin/core-quantity": "3.0.0-dev.116",
-    "@itwin/core-react": "3.0.0-dev.116",
-    "@itwin/presentation-common": "3.0.0-dev.116",
-    "@itwin/presentation-frontend": "3.0.0-dev.116",
-    "@itwin/webgl-compatibility": "3.0.0-dev.116",
+    "@bentley/imodelhub-client": "rc",
+    "@bentley/itwin-client": "rc",
```

Comment:
If you don't use this directly, can you try and drop it? I should have a PR later today to completely remove it.

Reply:
going to continue trimming deps in another pr